sunpy/net/dataretriever/sources/aia_synoptic.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+import re
+from datetime import datetime, timedelta
+from sunpy.net.dataretriever import GenericClient
+from sunpy.net import attrs as a
+import astropy.units as u
+from sunpy.net.dataretriever.client import QueryResponse
+
+__all__ = ["AIASynopticClient"]
+
+class AIASynopticClient(GenericClient):
+    """
+    A client for retrieving AIA synoptic data from JSOC.
+
+    This client retrieves synoptic AIA data from the following source:
+    https://jsoc1.stanford.edu/data/aia/synoptic/
+
+    The synoptic dataset includes lower-resolution (1k) images, with additional
+    image processing steps like downsampling and time integration.
+    """
+
+    baseurl = "https://jsoc1.stanford.edu/data/aia/synoptic/"
+    known_wavelengths = [94, 131, 171, 193, 211, 304, 335, 1600, 1700, 4500]
+
+    required = {a.Time, a.Instrument}
+    optional = {a.Sample, a.Level, a.Wavelength, a.ExtentType}
```

Comment:
```suggestion
    required = {a.Time, a.Instrument, a.ExtentType, a.Provider}
    optional = {a.Sample, a.Wavelength}
```
Maybe this would work, it's a lot to add for a query but it should make sure we only get the results from this client.